Automattic/jetpack
# Nav Redesign: Improve Plugins menu consistency

Fixes https://github.com/Automattic/dotcom-forge/issues/6940 (see also pfsHM7-YW-p2)

## Proposed changes:
    
In order to improve the consistency menu of the Plugins menu for Simple and Atomic sites, this PR:
- Registers a "Plugins > Marketplace" menu for both Simple and Atomic that links to the `/plugins/:site`
- Register a "Add New Plugin" menu for Simple sites that links to `plugins.php?page=wpcom-install-plugin`. In the future, `plugin-install.php` might redirect here (see pMz3w-k1d-p2).

<img width="168" alt="Screenshot 2024-05-29 at 13 18 59" src="https://github.com/Automattic/jetpack/assets/1233880/ea5bea55-1ade-4955-963b-455868fe6bbb">


### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
- Apply these changes to your WP.com classic site
- Go to either Calypso or WP Admin for that site
- Check the Plugins menu
- Make sure there is a new "Plugins Marketplace" submenu that links to `/plugins/:site`
- For Simple sites, make sure there is a "Add New Plugin" submenu that links to `plugins.php?page=wpcom-install-plugin` and check the splash screen is displayed
- Make sure the rest of the submenus work as expected
